grease-exe/src/Grease/Main.hs

```diff
@@ -961,6 +964,21 @@ macawRefineOnce la archCtx simOpts halloc macawCfgConfig memPtrTable setupHook a
                     )
                 ]
 
+addressCallBack ::
+  MM.MemWidth (MC.RegAddrWidth (MC.ArchReg arch)) =>
+  Proxy
+    arch ->
+  Handle ->
+  MM.MemSegmentOff (MC.RegAddrWidth (MC.ArchReg arch)) ->
+  IO ()
+addressCallBack _ hdl addr =
+  let js = renderJSON $ Load.memSegOffToSectionMemAddr addr
```

Comment:
Perhaps we should just export a function `MemSegmentOff -> JSON` from `Load`, rather than exporting `memSegOffToSectionMemAddr` + `SectionMemAddr`?